projects/OdroidXU3/bootloader/boot.ini

```diff
@@ -237,16 +225,16 @@ setenv hdmi_clk_amp_lvl     "31"
 #
 #  Hardkernrel default hdmi_tx_res = 0 (Source Termination OFF)
 #------------------------------------------------------------------------------
-setenv hdmi_tx_res      "0"
+# setenv hdmi_tx_res      "0"
 
 #------------------------------------------------------------------------------
 
-setenv hdmi_phy_control "hdmi_tx_amp_lvl=${hdmi_tx_amp_lvl} hdmi_tx_lvl_ch0=${hdmi_tx_lvl_ch0} hdmi_tx_lvl_ch1=${hdmi_tx_lvl_ch1} hdmi_tx_lvl_ch2=${hdmi_tx_lvl_ch2} hdmi_tx_emp_lvl=${hdmi_tx_emp_lvl} hdmi_clk_amp_lvl=${hdmi_clk_amp_lvl} hdmi_tx_res=${hdmi_tx_res} HPD=${HPD} vout=${vout}"
+# setenv hdmi_phy_control "hdmi_tx_amp_lvl=${hdmi_tx_amp_lvl} hdmi_tx_lvl_ch0=${hdmi_tx_lvl_ch0} hdmi_tx_lvl_ch1=${hdmi_tx_lvl_ch1} hdmi_tx_lvl_ch2=${hdmi_tx_lvl_ch2} hdmi_tx_emp_lvl=${hdmi_tx_emp_lvl} hdmi_clk_amp_lvl=${hdmi_clk_amp_lvl} hdmi_tx_res=${hdmi_tx_res} HPD=${HPD} vout=${vout}"
 
 
 # Load kernel, initrd and dtb in that sequence
 fatload mmc 0:1 0x40008000 KERNEL
-fatload mmc 0:1 0x44000000 exynos5422-odroidxu3.dtb
+fatload mmc 0:1 0x44000000 exynos5422-odroidxu4.dtb
```

Comment:
The xu4 boots fine with the xu3 dtb, I'm not sure about the opposite.

Reply:
Can you please test me this commit
https://github.com/Ntemis/Lakka-LibreELEC/commit/2af2aac7c4709e1d7cded4d36c04eb52f2cb2f48
Thanks

Reply:
Sure, but I'm getting a

    /home/kivutar/Lakka-LibreELEC/build.Lakka-OdroidXU3.arm-2.1-devel/toolchain/lib/gcc/armv7ve-libreelec-linux-gnueabi/6.2.0/../../../../armv7ve-libreelec-linux-gnueabi/bin/ld.gold: error: cannot find -lGLESv1_CM
collect2: error: ld returned 1 exit status

When linking RA